music_assistant/server/providers/opensubsonic/sonic_provider.py

```diff
@@ -427,6 +430,11 @@ async def _run_async(self, call: Callable, *args, **kwargs):
 
     async def resolve_image(self, path: str) -> bytes:
         """Return the image."""
+        # We assign None to the image ID if we don't have one and this used to be okay,
+        # however newer MA seems to be unhappy with this and will still ask us to resolve
+        # an image without a path.
```

Comment:
An image without a path ?! That is not good. If an item has no image, just dont set an image, dont set an empty path.

Reply:
I think we can actually drop this entirely, the empty path was a knock on from the ProviderMapping bug because I didn't see the failure from this method once that bug was fixed. I will drop this change from the patch and update.